LandSandBoat/server
# Fix visitant status removal setPos call to wait until after game in

**_I affirm:_**

## What does this pull request do?
Visitant effect loss on game in would try to setPos before the player existed.  This delays that action only on gameIn to be called in player.onGameIn.


## Steps to test these changes
Toggle GM, zone into Abyssea-Vunkerl, logout, login.  Player should remain in Vunkerl
